src/MasterXaction.cc

```diff
@@ -9,5 +9,5 @@
 #include "squid.h"
 #include "MasterXaction.h"
 
-InstanceIdDefinitions(MasterXaction, "MXID_");
+InstanceIdDefinitions(MasterXaction, uint64_t, "MXID_");
```

Comment:
I must have missed this in the original commit, but it looks like the ID prefix for master transactions was not logged much before so we can and should change it to be more readable, easier to remember/guess, and more consistent with the existing unique ID prefixes:

```suggestion
InstanceIdDefinitions(MasterXaction, uint64_t, "master");
```

For the record:

```
InstanceIdDefinitions(MasterXaction, "MXID_");
InstanceIdDefinitions(AsyncCall, "call");
InstanceIdDefinitions(AsyncJob, "job");
InstanceIdDefinitions(idns_query,  "dns");
InstanceIdDefinitions(HelperServerBase, "Hlpr");
InstanceIdDefinitions(Ipc::QueueReader, "ipcQR");
InstanceIdDefinitions(PeerSelector, "PeerSelector");
InstanceIdDefinitions(MemBlob, "blob");
InstanceIdDefinitions(SBuf, "SBuf");
```